core/src/components/modal/modal.tsx

```diff
@@ -820,7 +881,20 @@ export class Modal implements ComponentInterface, OverlayInterface {
         {mode === 'ios' && <div class="modal-shadow"></div>}
 
         <div role="dialog" class="modal-wrapper ion-overlay-wrapper" part="content" ref={(el) => (this.wrapperEl = el)}>
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && (
+            <button
+              class={{
+                'modal-handle': true,
+                'modal-handle--clickable': isHandleCycle,
```

Comment:
Shouldn't the handle always be clickable? Users can double tap here to drag and adjust the size of the sheet manually:

https://user-images.githubusercontent.com/2721089/176693672-56bc2053-2b86-41f8-bc11-90629233190e.mp4

I know you can drag the entire header, but the handle was added as a cue for how to expand/collapse the sheet. Even if the handle cycle is 'none' you should still be able to click and drag on it.

Reply:
This class name may be misleading. Before this feature the handle did not have a `cursor: pointer` styling. This class conditionally applies that styling if clicking the handle advances to the next breakpoint. 

Should we show the cursor pointer style always or rename this class to better explain it is applying cursor styles?